code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -83,8 +83,31 @@
 	addtimer(CALLBACK(src, .proc/show_playstyle), 5)
 
 /mob/living/silicon/robot/proc/create_modularInterface()
-	if(!modularInterface)
+	// TG ORIGINAL BEGIN
+	//if(!modularInterface)
+	//	modularInterface = new /obj/item/modular_computer/tablet/integrated(src)
+	// TG ORIGINAL END
+	
+	//SKYRAT EDIT ADDITION BEGIN
+	//Creating a certain cyborg tablet depending on his species
+	if (model.name == "Default")
```

Comment:
This is causing runtimes, model is a null at this point, which makes me theorise that this is called before setting up the model for the borg

Reply:
Okay. I will do a separate procedure for this purpose.

Reply:
I restored the standard tablet creation and added a procedure for installing programs